### DISCUSSION TITLE
LXQt 1.3 for Debian

### DISCUSSION THREAD

**Discussion by severusseptimius** [lxqt/lxqt#2430] [Show and tell]
Hello! It's been a while since I last packaged LXQt for Debian. Last time I learnd a lot about both of them and it's been a wonderful experience. So I'm happy to announce that I've done it again. The packages for LXQt 1.3 can be found here: [https://github.com/severusseptimius/severusseptimius.github.io](https://github.com/severusseptimius/severusseptimius.github.io) .

This time it is much easier for users to install it since it is now configured as a debian repository. Also, the transition to the official packages when they arrive should be a lot more smoother.

**Comment by stefonarch** [comment]
Good to hear, thanks a lot!

EDIT: installed on the main PC with debian bookworm, all good.

**Comment by Attigliuzzo** [comment]
I decide to install manually on my Bookworm all packages. I know it is simpler and faster using github as repo but......

I found out that the package lxqt-session depends on libproc2 version 2:4.0.3 (Sid), Bookworm is still on  2:4.0.2.

![immagine](https://github.com/lxqt/lxqt/assets/42100711/5c9ec728-5efd-483c-ac7b-5ade7285801d)

Anyway i installed everything. 
Good job man

**Reply by stevenpusser** [reply]
Hey, packager for a Debian derivative here!  We have a user request to port LXQT 1.3 to our upcoming Bookworm-based release's repositories. Do you have any of your debianized source files for the debs we could use for that? TIA

**Reply by severusseptimius** [reply]
You can find them here: https://github.com/severusseptimius/lxqt13-bookworm

**Reply by stevenpusser** [reply]
Much thanks!  After the Bookworm builds, I'll see what happens for Bullseye.

**Comment by stevenpusser** [comment]
I got them to build even on Bullseye (MX 21) with some additional backports from Bookworm, such as libprocps2-dev, but I had to add an additional build-depend for both Bookworm and Bullseye on this package for them to build in "clean rooms" provided by pbuilder:  

```
dg-desktop-portal-lxqt (0.4.0-0.1~mx21+1) mx; urgency=medium

  * Rebuild for MX Linux.
  * Add qtbase5-private-dev to build-deps to prevent FTBFS.
```

"FTBFS" is "Failure to Build From Source", often used by package maintainers and something I get myself sadly way too often.

They are in the MX test repository for users to try out before we can move them to our main repositories.

Our MX test repos are like Debian backports, so aren't seen as automatic upgrades. I'm going to do a customized "lxqt-core" metapackage which will depend on 1.3.0, so installing that will upgrade the whole schmeer.

**Reply by stevenpusser** [reply]
Hey, I can also use the openSUSE Build Service (OBS) to build the packages from source for multiple Debian architectures, and also automatically create repositories for them. That would kind of deprecate this github repo, though...
